client/components/cart-summary-item.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+export default function CartSummaryItem(props) {
+  const revPrice = ((props.item.price * 0.01).toFixed(2));
+  return (
+    <div className="row mb-3 mx-0">
+      <div className="card mx-auto">
+        <div className="row">
+          <img
+            src={props.item.image}
+            className="card-img-top col-4"
+            alt={props.item.name}
+            style={{ objectFit: 'contain', height: '300px' }} />
```

Comment:
Good practice is to move styling into your CSS and add a class here, that way it's easily updated without having to find the specific image in your code.